draft-ietf-add-ddr.md

```diff
@@ -186,14 +187,18 @@ validating designation as defined in {{verified}}.
 # Discovery Using Resolver IP Addresses {#bootstrapping}
 
 When a DNS client is configured with an Unencrypted DNS Resolver IP address, it
-SHOULD query the resolver for SVCB records for the name "resolver.arpa" before
-making other queries, in order to use Encrypted DNS for all other queries,
-if possible. Specifically, the client issues a query for `_dns.resolver.arpa.`
-with the SVCB resource record type (64) {{I-D.ietf-dnsop-svcb-https}}.
+SHOULD query the resolver for SVCB records of a service with a scheme of "dns" and
+an Authority of "resolver.arpa" before making other queries. This allows the client
+to switch to using Encrypted DNS for all other queries, if possible. Specifically,
+the client issues a query for `_dns.resolver.arpa.` with the SVCB resource record type
+(64) {{I-D.ietf-dnsop-svcb-https}}.
 
 Because this query is for an SUDN, which no entity can claim ownership over,
 the ServiceMode SVCB response MUST NOT use the "." value for the TargetName. Instead,
 the domain name used for DoT/DoQ or used to construct the DoH template MUST be provided.
+This ensures that different designated resolver configurations can be correctly
+associated with IP addresses in A and AAAA records. As such, clients MUST NOT
+perform speculative A and AAAA queries for "resolver.arpa".
```

Comment:
This is the relevant text in the svcb-http draft:

>   To improve performance when using a non-conforming recursive
   resolver, clients SHOULD issue speculative A and/or AAAA queries in
   parallel with each SVCB query, based on a predicted value of
   TargetName (see Section 11.2).

Because resolver.arpa is never valid for A/AAAA records in the first place, we could be even more specific and avoid having this esoteric reference to the draft by removing the speculative term (you MUST NOT do those because you MUST NOT issue the A/AAAA queries for the name ever anyway).